light-rpc/metrics.csv

```diff
@@ -0,0 +1,102 @@
+32.336399,33.518671,1.1822719999999975
+31.660123,29.812578,1.8475450000000002
+36.337776,32.448687,3.8890889999999985
+32.133218,30.358182,1.775036
+36.163395,33.847729,2.3156660000000002
+35.397548,31.3822,4.0153479999999995
+33.306766,30.670772,2.6359940000000037
+32.03427,30.506387,1.5278829999999992
+32.17269,33.780526,1.607835999999999
+31.701694,30.523642,1.178052000000001
+33.651125,32.840389,0.8107359999999986
+33.866456,29.914559,3.951896999999999
+32.153237,30.049765,2.1034719999999965
+31.627936,31.31393,0.3140059999999991
+31.906213,29.959911,1.9463019999999993
+34.227758,34.681657,0.45389899999999983
+36.597028,35.265055,1.331973000000005
+36.227975,30.336292,5.8916830000000004
+31.868848,29.992122,1.8767260000000014
+31.808913,31.062966,0.745947000000001
+31.411289,30.331229,1.0800599999999996
+31.950963,31.286122,0.6648410000000027
+32.574338,36.078786,3.5044480000000036
+37.410406,33.64087,3.769536000000002
+35.614952,30.496037,5.118915000000001
+32.60809,30.270275,2.3378149999999955
+31.370894,30.863056,0.5078379999999996
+35.228478,30.153974,5.074504000000001
+32.112327,35.234183,3.121856000000001
+30.933101,31.751531,0.8184299999999993
+31.679448,30.41198,1.267468000000001
+37.441329,32.192367,5.248962000000006
+34.446785,30.735077,3.711707999999998
+34.328816,35.730798,1.4019819999999967
+34.498321,41.689451,7.191130000000001
+37.57391,36.401192,1.1727179999999962
+36.16693,40.524709,4.357779000000001
+42.326964,35.942121,6.3848429999999965
+33.899098,31.086518,2.8125800000000005
+47.382627,36.502009,10.880617999999998
+40.522043,34.387876,6.134166999999998
+35.124356,35.107372,0.016984000000000776
+36.31238,30.796939,5.515440999999999
+31.798349,36.001376,4.203026999999999
+32.288089,32.999921,0.7118320000000011
+32.854353,30.40379,2.4505630000000025
+34.809454,31.290031,3.5194230000000033
+34.844599,36.887977,2.043377999999997
+34.819599,29.878673,4.9409259999999975
+33.081473,39.864683,6.783209999999997
+33.961698,41.960407,7.998708999999998
+34.088724,36.386772,2.2980480000000014
+33.126125,31.508729,1.617396000000003
+34.588443,31.968234,2.620208999999999
+35.333871,29.98058,5.353291000000002
+33.863691,31.340129,2.523562000000002
+32.928475,41.983848,9.055373000000003
+35.719844,35.08662,0.6332239999999985
+32.400489,32.645156,0.24466699999999975
+34.519766,32.019861,2.4999049999999983
+35.78428,31.742363,4.0419170000000015
+35.652941,30.697814,4.955126999999997
+35.358074,33.227494,2.130580000000002
+33.605454,37.692786,4.087331999999996
+34.502848,35.205536,0.702688000000002
+32.955227,30.928542,2.0266850000000005
+34.368056,30.702471,3.6655850000000036
+34.69397,31.256388,3.437581999999999
+34.355047,37.348667,2.99362
+35.543551,37.566298,2.0227470000000025
+33.739997,36.60322,2.863222999999998
+34.2413,35.888405,1.6471049999999963
+39.02204,39.16363,0.14159000000000077
+38.457204,36.360513,2.096691
+40.040714,34.705837,5.334876999999999
+38.79393,37.941486,0.8524440000000055
+42.978491,37.063746,5.914744999999996
```

Comment:
please always include a header column.

we had a look at these results and they don't match our experience, it looks like everything got averaged